src/python/dxpy/bindings/apollo/dataset.py

```diff
@@ -0,0 +1,120 @@
+from dxpy import DXHTTPRequest
+from dxpy.bindings import DXRecord
+
+
+class Dataset(DXRecord):
+    def __init__(self, record_id, project_id):
+        self.record_id = record_id
+        self.project_id = project_id
+        self.visualize_info = None
+        self._dx_record_obj = None
+        self.dx_dataset_descriptor = None
+        self.assays_list = None
+        self.assay_names_list = None
+
+    def get_visualize_info(self):
+        if self.visualize_info is None:
+            self.visualize_info = DXHTTPRequest(
+                "/" + self.record_id + "/visualize",
+                {"project": self.project_id, "cohortBrowser": False},
+            )
+        return self.visualize_info
+
+    @property
+    def vizserver_url(self):
+        vis_info = self.get_visualize_info()
+        return vis_info.get("url")
+
+    def get_dx_record(self):
+        if self._dx_record_obj is None:
+            self._dx_record_obj = DXRecord(self.record_id)
+        return self._dx_record_obj
+
+    @property
+    def dx_record_obj(self):
+        return self.get_dx_record()
+
+    @property
+    def record_details(self):
+        return self.dx_record_obj.get_details()
+
+    @property
+    def record_descriptor(self):
```

Comment:
This could be misleading, because `descriptor` is the `file` that is linked to the dataset. I'd suggest simply calling it `describe` but then again if the superclass is already initialized, then it already contains `describe` info.

Reply:
`describe` might be confused with the results of `/describe` call as the `descriptor` json contains different info

Reply:
@jkotrs Yes, but it's exactly what this class is doing, it's returning the result of the `/describe` call. This is why I suggested calling it describe.